configs/src/deployments/osx-ens-domains.ts

```diff
@@ -0,0 +1,14 @@
+import {SupportedNetworks} from '../networks';
+import {NetworkDomains, NetworkDomain} from './types';
+
+export const commonDomain: NetworkDomain = {
+  daoEns: 'dao.eth',
+  pluginEns: 'plugin.dao.eth',
+};
+
+export const exceptionalDomains: NetworkDomains = {
+  [SupportedNetworks.SEPOLIA]: {
+    daoEns: 'aragon-dao.eth',
+    pluginEns: 'plugin.aragon-dao.eth',
+  },
+};
```

Comment:
Nice, but i am wondering if we can have a better naming here?
any ideas @Michael-A-Heuer?

Reply:
Yeah, I struggled to find better names. 
I also thought about `defaultDomain` and maybe `specialDomain`. But liked more `common` and `exceptional`.

Reply:
maybe `osxCommonDomain` | `osxDefaultDomain` and `osxAlternativeDomain` | `osxExceptionalDomain` | `osxSpecialDomain` wdyt?

Reply:
> But liked more `common` and `exceptional`.

I also prefer those